security.md

```diff
@@ -0,0 +1,76 @@
+discover and report vulnerabilites in nix-bitcoin or its upstream dependencies.
+Rewards are paid out as percentages of the total fund, rather than as fixed
+amounts.
+
+```
+Insert address here
+```
+
+The following nix-bitcoin developers each hold one key to the multisig address
+and collectively form the nix-bitcoin developer quorum:
+
+- Jonas Nick (0xB1A70E4F8DCD0366) [@nickler:nixbitcoin.org](https://matrix.to/#/@nickler:nixbitcoin.org)
+- Erik Arvstedt (0x33312B944DD97846) [@erikarvstedt:matrix.org](https://matrix.to/#/@erikarvstedt:matrix.org)
+- nixbitcoindev (0xDD11F9AD5308B3BA) [@nixbitcoindev:nixbitcoin.org](https://matrix.to/#/@nixbitcoindev:nixbitcoin.org)
+
+## Eligible Vulnerabilities
+
+The following types of vulnerabilities are eligible for reward payouts, to the
+exclusion of all other security vulnerabilities.
+
+| Type | Description | Examples |
+| :-: | :-: | :-: |
+| Outright Vulnerabilites | Vulnerabilities in nix-bitcoin specific tooling (except CI tooling) | `netns-exec`, `nodeinfo`, improper release signature verification through `fetch-release` |
+| Violations of PoLP | nix-bitcoin services are given too much privilege over the system or unnecessary access to other nix-bitcoin services, or one of the nix-bitcoin isolation measures is incorrectly implemented | `netns-isolation` doesn't work, spark-wallet has access to bitcoin rpc interface or files | 
+| Vulnerabilities in Dependencies | A vulnerability in any dependency of a nix-bitcoin installation with a configuration consisting of any combination of the following services (bitcoind, clightning, lnd, electrs, joinmarket, btcpayserver, liquidd) | Compromised NixOS expression pulls in malicious package, JoinMarket pulls in a python dependency with a known severe vulnerability |
+| Bad Documentation | Our documentation suggests blatantly insecure things | `install.md` tells you to add our ssh keys to your root user |
+| Compromise of Signing Key | Compromise of the nix-bitcoin signing key, i.e., `0xB1A70E4F8DCD0366` | Leaking the key, managing to sign something with it |
+
+## Payout Percentage
+
+If a vulnerability or any combination of a number of vulnerabilities that meet
+the above-described eligibility requirements can be exploited using a
+demonstrable attack scenario on nix-bitcoin users, researchers are eligible for
+an increased payout percentage (in comparison to the 10 % baseline nominal
+reward payout) corresponding to the final outcome of the attack scenario.
+
+| Outcome | Description | Nominal Payout Percentage\* |
+| :-: | :-: | :-: |
+| Loss of Funds | If a demonstrable exploit provides a direct path for stealing or destroying user's funds | 50 % |
+| Loss of Privacy | If a demonstrable exploit provides a direct path for exfiltrating sensitive information or otherwise attributing a user's real world identity to his nix-bitcoin node or funds held/managed thereon without him specifically opting-in to this | 25 % |
```

Comment:
We don't mention the secure-node preset anywhere, but without Tor isn't enforced and this should be easy.